### DISCUSSION TITLE
Local offset, difficult to use

### DISCUSSION THREAD

**Discussion by MikkelCornelius** [ppy/osu#27699] [Ideas] [answered]
The slider for local offset is very sensitive, and there is nothing for it to snap to. If you mess with the offset, it's practically impossible to get it back to 0.0 ms. 

Can we make it so the local offset snaps to every 5 ms or smth, because it's really annoying when the game constantly tells me I'm playing with 0.1 offset.

**Comment by BlockHead007** [comment, marked as answer]
you can get 0.0 actually

![osu!_KdgTTnndsi](https://github.com/ppy/osu/assets/25553854/09b837a3-816a-46bd-a782-54b3d4e38695)

and you can hold shift to "snap to every 5 ms"
